### PR TITLE
Add CSS Minification Example

### PR DESCRIPTION
**Pull request by Bhavesh27**
Resolves #26

### CODE REVIEW

**Comment by kabirbaidhya** [comment]
Hey @Bhavesh27, Thanks for contributing. 
However, the purpose of these examples were to demonstrate the use of [webpack-file-preprocessor-plugin](https://github.com/kabirbaidhya/webpack-file-preprocessor-plugin) to pre-process assets - css in this case. But you actually haven't used `webpack-file-preprocessor-plugin` at all. And have used other plugins to do the same which doesn't make sense.

Please refer to other [examples](https://github.com/kabirbaidhya/webpack-file-preprocessor-plugin/tree/master/examples) and see how they're doing and make changes accordingly.

File: examples/css-minifier/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      },
+      canPrint: true
+    })],
+  },
+  plugins: [
+    new MiniCssExtractPlugin({
+      filename: '[name].[hash].css',
+      chunkFilename: '[id].[hash].css',
+    }),
+  ],
```
Comment (kabirbaidhya) [inline examples/css-minifier/webpack.config.js]: Try using the `webpack-file-preprocessor-plugin` here whose use case was to be denostrated. Please consider checking this [example](https://github.com/kabirbaidhya/webpack-file-preprocessor-plugin/blob/master/examples/html-minifier/webpack.config.js).

File: README.md [hunk 2]
```diff
@@ -62,10 +62,61 @@ module.exports = {
...
+          {
+            loader: 'css-loader',
+            options: {}
+          }
+        ],
+      },
+    ],
+  },
+};
+```
```
Comment (kabirbaidhya) [inline README.md]: Please remove this example. You've already put a link to the example project under `Examples` which is enough.

File: examples/css-minifier/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+```
+$ yarn build
+```
+
+![Building](misc/screenshot.png)
+
+## Configuration
+
+Check the webpack configuration [here](webpack.config.js).
```
Comment (kabirbaidhya) [inline examples/css-minifier/README.md]: Please update it as per your new example title, screenshot, etc. - everything here looks like a duplicate of previous example to me.

File: examples/css-minifier/webpack.config.js (hunk 1 above)
Comment (Bhavesh27) [reply examples/css-minifier/webpack.config.js]: sorry i misunderstood the task. Will update the PR.

File: examples/css-minifier/README.md (hunk 3 above)
Comment (Bhavesh27) [reply examples/css-minifier/README.md]: i missed this one out. Will do

**Comment by Bhavesh27** [comment]
Done

**Pull request merged**